protocol/contracts/lpp/src/contract/mod.rs

```diff
@@ -29,6 +29,8 @@ mod rewards;
 #[cfg(feature = "migration")]
 const CONTRACT_STORAGE_VERSION_FROM: VersionSegment = 0;
 const CONTRACT_STORAGE_VERSION: VersionSegment = 1;
+const CONTRACT_VERSION: SemVer = package_version!();
+const COMPLETE_VERSION: Version = version!(CONTRACT_STORAGE_VERSION, CONTRACT_VERSION);
```

Comment:
here the two issues are still present .... `lpp` was not in the find/replace scope? ;-)

Reply:
Somehow it wasn't ;) Will correct that for the next time.